### PR TITLE
feat: Firewall policy rule with resource manager tag

### PR DESCRIPTION
**Pull request by Samir-Cit**
Hello folks.

This PR is regarding the usage of resource manager tags for firewall policy rules and on VMs.

Two new resource manager tags created for the two firewall rules that were refactored, allowing IAP SSH and IAP RDP. The peering network project was used as an example, creating a new sub network on the peered network to be used by the VM.

### CODE REVIEW

File: 4-projects/business_unit_1/development/outputs.tf [hunk 1]
```diff
@@ -98,3 +98,13 @@ output "bucket" {
   description = "The created storage bucket"
```
Comment (amandakarina) [inline 4-projects/business_unit_1/development/outputs.tf:98]:
```suggestion
  description = "The created storage bucket".
```

File: test/integration/projects/projects_test.go [hunk 2]
```diff
@@ -221,8 +222,21 @@ func TestProjects(t *testing.T) {
...
+							assert.Subset(listRoles, peering_project_sa_roles, fmt.Sprintf("service account %s should have project level roles", sharedCloudBuildSA))
+
 							peering := gcloud.Runf(t, "compute networks peerings list --project %s", projectID).Array()[0]
 							assert.Contains(peering.Get("peerings.0.network").String(), tt.baseNetwork, "should have a peering network")
+
+							peering_subnetwork_self_link := projects.GetStringOutput("peering_subnetwork_self_link")
+							peering_subnetwork_self_link_splitted := strings.Split(peering_subnetwork_self_link, "/")
+							peering_subnetwork_name := peering_subnetwork_self_link_splitted[len(peering_subnetwork_self_link_splitted)-1]
+							subnet := gcloud.Run(t, fmt.Sprintf("compute networks subnets describe %s --project %s --region us-central1", peering_subnetwork_name, projectID))
+							assert.Equal("PRIVATE", subnet.Get("purpose").String(), "Purpose should be PRIVATE")
```
[the remark this answers is not in the document]
Comment (Samir-Cit) [reply test/integration/projects/projects_test.go:241]: Created some tests for the firewall policy rules.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**